deploy/kubernetes/helm/che/values.yaml

```diff
@@ -66,12 +66,20 @@ global:
     runAsUser: 1724
     fsGroup: 1724
   postgresDebugLogs: false
+  cheDevfileRegistryUrlFormat: "che-devfile-registry-%s.%s"
+  chePluginRegistryUrlFormat: "che-plugin-registry-%s.%s"
 
-che: {}
-#  workspace:
+che:
+  workspace: {}
 #    devfileRegistryUrl: "https://che-devfile-registry.openshift.io/"
 #    pluginRegistryUrl: "https://che-plugin-registry.openshift.io/v3"
 
+cheDevfileRegistry:
+  deploy: true
```

Comment:
Couldn't we change this to `external: false`, to be consistent with what was decided for the operator (and what is already in place for the Keycloak case )?
cc @l0rd

Reply:
If this is the way how we're doing that in operator, I agree. Do you know how to do `NOT` in `requirements.yaml` `condition:` ? https://github.com/sparkoo/che/blob/helm-chart-deploy-registries/deploy/kubernetes/helm/che/requirements.yaml#L14 I can't find anything how to achieve that and all my experiments are failing so far :/

Reply:
> If this is the way how we're doing that in operator, I agree

it is

> Do you know how to do NOT in requirements.yaml condition ?

TBH I don't know. Not a Helm expert, but what if you use an `else` as mentioned here: https://github.com/helm/helm/blob/master/docs/chart_template_guide/control_structures.md#ifelse

Reply:
Sorry, I didn't read the question correctly: if/else has nothing to do with your question it seems. So no, I don't know. Maybe using a calculated value that could be defined as the opposite of `external` ?

Reply:
as far as I know, `requirements.yaml` is just clean yaml, no interpolation. And I believe all values there must be taken from `values.yaml`, which is the again just clean yaml. For keycloak we're doing check for `true` here https://github.com/eclipse/che/blob/master/deploy/kubernetes/helm/che/requirements.yaml#L18 `global.cheDedicatedKeycloak`. So we might go with `deploy=true` way, just for technology conventions sake here.

Reply:
Yeah, it seems that you will have to do the same as for Keycloak.

I assume that CheCtl should do the same as the operator though, to maintain some sort of consistency.
@l0rd do you confirm ?

Reply:
You mean that there will be some inconsistency amongst helm and operator in the semantic of the parameter right? Yes it doesn't seam like a big deal and we can let chectl manage it